### ISSUE TITLE
NameError: global name 'root_dir' is not defined

### ISSUE THREAD

**Issue by scaidermern** [mapillary/mapillary_tools#351]
Adding `--summarize` to the upload tool produces the following error message:

```
$ ~/.local/bin/mapillary_tools upload --verbose --advanced --skip_subfolders --number_threads 1 --max_attempts 10 --import_path "/media/foo/" --summarize
All images have already been uploaded
Done uploading 0 images.
Traceback (most recent call last):
  File "/home/carl/.local/bin/mapillary_tools", line 76, in <module>
    command.run(args)
  File "/home/carl/.local/lib/python2.7/site-packages/mapillary_tools/commands/upload.py", line 52, in run
    post_process(**({k: v for k, v in vars_args.iteritems()
  File "/home/carl/.local/lib/python2.7/site-packages/mapillary_tools/post_process.py", line 161, in post_process
    import_path, skip_subfolders)
  File "/home/carl/.local/lib/python2.7/site-packages/mapillary_tools/processing.py", line 680, in get_duplicate_file_list
    duplicate_file_list.extend(os.path.join(os.path.abspath(root_dir), file) for file in os.listdir(root_dir) if file.lower().endswith(
NameError: global name 'root_dir' is not defined
```

**Comment by MSolodzhuk** [comment]
if not root_dir:
        root_dir = import_path

**Comment by MSolodzhuk** [comment]
added root_dir to function

**Comment by fgouget** [comment]
This looks like the same issue I submitted a fix for in January 2019 (rebased since):
https://github.com/fgouget/mapillary_tools/commit/d9dbac40616c4dd324b06dd2f171dabd27a05265

I wish Mapillary would use it or come up with their own fix so people don't keep running into it.

**Comment by ptpt** [comment]
Thanks all! It's fixed in 0.6.0 (Python3 only).
